_studio/mfx_lib/encode_hw/mjpeg/src/mfx_mjpeg_encode_vaapi.cpp

```diff
@@ -195,47 +238,8 @@ mfxStatus VAAPIEncoder::QueryBitstreamBufferInfo(mfxFrameAllocRequest& request)
 mfxStatus VAAPIEncoder::QueryEncodeCaps(JpegEncCaps & caps)
 {
     MFX_CHECK_NULL_PTR1(m_core);
```

Comment:
Is there any possibility to call this function when ```m_caps``` is not initialized yet? or ```m_core``` check guaranties that it won't happen?